pkg/addon/template.go

```diff
@@ -198,6 +206,11 @@ type CNIPlugin struct {
 	Version string
 }
 
+type VsphereConfiguration struct {
+	// StoragePolicy is the storage policy to use for vsphere csi addon
+	StoragePolicy string
+}
```

Comment:
Where do we use this struct?

Reply:
It's garbage left from a previous implementation that I forgot to delete. Will remove it.